### DISCUSSION TITLE
`Module parse failed: Unexpected token` `??`

### DISCUSSION THREAD

**Discussion by lanluu** [themesberg/flowbite-react#561] [Q&A] [answered]
Hi all,

I followed you tutorial to install Flowbite https://flowbite.com/docs/getting-started/react/.

By importing any Component into my React App e.g. `import { Button } from 'flowbite-react';`, I receive the following error: 

```
Failed to compile.

./node_modules/flowbite-react/lib/esm/components/FormControls/HelperText.js 17:21
Module parse failed: Unexpected token (17:21)
You may need an appropriate loader to handle this file type.
|     className: classNames(theme.base, theme.colors[color], className)
|   }, props, {
>     children: value ?? children ?? ''
|   }));
| };
```
Could you please help me to resolve such issue? Thank you very much!

**Comment by tulup-conner** [comment, marked as answer]
Hi there, this is an old post, but to answer your question, **you are having this issue because the version of ECMAScript you are "transpiling" to in your environment is too old to support `??`**. 

It's referred to as the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing).

You'll need to use a newer "language version" of JavaScript to fix this issue, like ES2015.

If you're using TypeScript, you can achieve this with [`tsconfig.json`](https://www.typescriptlang.org/tsconfig#lib).
